modules/cloudbuild/cloudbuild_builder/Dockerfile

```
@@ -23,6 +23,7 @@ ENV ENV_TERRAFORM_VERSION=$TERRAFORM_VERSION
 ENV ENV_TERRAFORM_VERSION_SHA256SUM=$TERRAFORM_VERSION_SHA256SUM
 
 RUN apt-get update && \
+    apt-get -y install google-cloud-sdk-terraform-tools && \
     apt-get -y install curl jq unzip git ca-certificates && \
```

I'd suggest doing as a single apt-get transaction.

```suggestion
    apt-get -y install curl jq unzip git ca-certificates google-cloud-sdk-terraform-tools && \
```

---

great idea.
done